### PR TITLE
Update oref0-setup.md

### PR DESCRIPTION
**Pull request by Bender1061**
alternative options to add aliases in .bash_profile

### CODE REVIEW

File: docs/docs/walkthrough/phase-2/oref0-setup.md [hunk 1]
```diff
@@ -99,3 +99,23 @@ When you decide to enable the new loop in cron, follow the log file (and watch N
...
+```
+alias autosens-loop=" tail -f /var/log/openaps/autosens-loop.log"
+alias autotune=" tail -f /var/log/openaps/autotune.log"
+alias network=" tail -f /var/log/openaps/network.log"
+alias ns-loop=" tail -f /var/log/openaps/ns-loop.log"
+alias pump-loop=" tail -f /var/log/openaps/pump-loop.log"
+alias urchin-loop=" tail -f /var/log/openaps/pump-loop.log"
+alias cgm-loop=" tail -f /var/log/openaps/cgm-loop.log
+alias xdrip-loop=" tail -f /var/log/openaps/xdrip-loop.log
+alias cgm-dexusb-loop=" tail -f /var/log/openaps/cgm-dexusb-loop.log
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-2/oref0-setup.md]: Why the spaces before each tail command?

File: docs/docs/walkthrough/phase-2/oref0-setup.md [hunk 2]
```diff
@@ -99,3 +99,23 @@ When you decide to enable the new loop in cron, follow the log file (and watch N
...
+so run `nano .bash_profile`
+
+add these lines or edit them to whatever you would like
+```
+alias autosens-loop=" tail -f /var/log/openaps/autosens-loop.log"
+alias autotune=" tail -f /var/log/openaps/autotune.log"
+alias network=" tail -f /var/log/openaps/network.log"
+alias ns-loop=" tail -f /var/log/openaps/ns-loop.log"
+alias pump-loop=" tail -f /var/log/openaps/pump-loop.log"
+alias urchin-loop=" tail -f /var/log/openaps/pump-loop.log"
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-2/oref0-setup.md]: This looks like a duplicate of `l` and `pump-loop`

File: docs/docs/walkthrough/phase-2/oref0-setup.md (hunk 1 above)
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-2/oref0-setup.md]: Also, we might want to make them all `tail -n 100 -F` to provide 100 lines of recent context...

**Comment by scottleibrand** [comment]
I also wonder if we could add these to oref0-setup instead of making people do them manually?

**Comment by Bender1061** [comment]
Fixed the issues, and I'll look into adding it to oref0-setup

File: scripts/quick-src.sh [hunk 3]
```diff
@@ -28,7 +28,7 @@ cd ~/src && \
     (cd openaps-contrib && \
         sudo python setup.py develop
     )
-    git clone -b dev git://github.com/openaps/oref0.git || \
+    git clone -b Del-fut-Data git://github.com/Bender1061/oref0.git || \
```
Comment (scottleibrand) [inline scripts/quick-src.sh]: Will want to remove this for merging to dev.

Comment (scottleibrand) [inline scripts/quick-src.sh]: (Or maybe you intended these commits to be on a new branch, not one that already had an open PR?)

**Comment by Bender1061** [comment]
put the stuff in the oref0-setup instead

**Pull request closed without merging**